src/pages/index.js

```diff
@@ -36,26 +133,110 @@ function handleImageClick(name, link) {
   imagePopup.open({ name, link });
 }
 
-// New PopupWithForm instances (userinfo integration change)
+// New PopupWithForm instances (userinfo integration change step 3)
 const editProfilePopup = new PopupWithForm("#profile-edit-modal", (data) => {
-  userInfo.setUserInfo({ name: data.title, job: data.description });
-  editProfilePopup.close(); //closes popup after editing profile
+  const submitButton = editProfilePopup._form.querySelector(".modal__button");
```

Comment:
It’s better to find **constant DOM elements** only 1 time at the top of the file so as not to waste resources on searching them again and again when you call a method (function) because searching is a very **hard** operation for the browser engine


All submit buttons should be found only once outside the functions. Please, add correct names for them to distinquish them from each other


Or you can make a special method inside `PopupWithForm` to get the button